enterprise/templates/enterprise/enterprise_course_enrollment_page.html

```diff
@@ -8,7 +8,7 @@
 
 {% block contents %}
   <div class="enterprise-container">
-    <div class="row">
+    <div class="row no-gutters">
```

Comment:
Curious about the need for `.no-gutters`? Is the column spacing coming from custom styles throughout these stylesheets? If so, should it be?

Reply:
Welllll... yes and no. There's one page in which adding this made the logo line up with the header logo and one in which it made them misaligned. But it made things a lot nicer on mobile? I'm open to removing it, but wary of falling into endlessly tweaking the css. I think we should do a design review like we did for admin portal, and open a bunch of tickets so we can tweak the css with more purpose and direction. 😉